src/actions/guides/testing/introduction.cr

```diff
@@ -0,0 +1,75 @@
+class Guides::Testing::Introduction < GuideAction
+  guide_route "/testing/introduction"
+
+  def self.title
+    "Intro to Testing"
+  end
+
+  def markdown : String
+    <<-MD
+    ## Getting Started
+
+    All of your tests will go in the `spec/` directory located at the top level of your application.
+    When you generate your Lucky project, there are a few files generated for you to help make testing
+    easier.
+
+    > The files generated will be different based on Full/Api and/or Authentication generated.
+
+    ### spec_helper.cr
+
+    The `spec/spec_helper.cr` file will require all of the necessary files and configuration your testing
+    suite will need to run. Every time you add a new spec file for testing, you will need to require this file
+    first. Read more on [testing in Crystal](https://crystal-lang.org/reference/guides/testing.html).
+
+    ## Flows
+
+    If you need to test your HTML views and/or the interactivity of your site (e.g. a user clicking a button to login),
+    you will create your flow specs in this directory.
+
+    > If you generated authentication for your app, you will have a few flow specs added in for you!
+
+    ## Setup
+
+    This directory is used to setup all of your testing. Any additional setups needed before your spec suite runs
```

Comment:
Yeah, see, same here. You say "test suite", then "spec suite". I'm leaning toward saying "spec suite" since it's in a spec directory. But what are your thoughts?

Reply:
I'm fine with either as long as we include some kind of explanation in the intro for people less familiar. I kind of like test suite just because spec suite sounds weird to me but that may just be me